apps/data-extraction/src/extractors/submarine-parts.extractor.ts

```diff
@@ -0,0 +1,59 @@
+import { AbstractExtractor } from '../abstract-extractor';
+import { map, switchMap, tap } from 'rxjs/operators';
+import { XivapiEndpoint } from '@xivapi/angular-client';
+
+export class SubmarinePartsExtractor extends AbstractExtractor {
+
+  protected doExtract(): any {
+    const parts = {};
+
+    this.getAllPages(this.getSearchEndpointWithQuery({
+      indexes: XivapiEndpoint.Item,
+      columns: 'ID,AdditionalData',
+      string_column: 'FilterGroup',
+      string: '36',
+      string_algo: 'match'
```

Comment:
You should use filters instead, this is the query string fragment for it: `filters=FilterGroup=36`, matching a number using a fulltext search is not the way to go for this one :)